### ISSUE TITLE
TokenRecognitionError

### ISSUE THREAD

**Issue by lneumeier** [Riverside-Software/sonar-openedge#752]
Hello,

I'm getting lot's of errors with the following message at the beginning:

```
INFO: Adding DLC directory 'D:\Progress\dlc121x64' to PROPATH
line 1110:3 token recognition error at: '-'
line 1111:3 token recognition error at: '-'
line 1112:3 token recognition error at: '-'
line 1113:3 token recognition error at: '-'
line 1114:2 token recognition error at: '-'
line 185:3 token recognition error at: '-'
line 1890:3 token recognition error at: '-'
line 1891:3 token recognition error at: '-'
line 1892:3 token recognition error at: '-'
...
line 10108:12 token recognition error at: '-'
line 10138:12 token recognition error at: '-'
line 10171:12 token recognition error at: '-'
INFO: Load project repositories
INFO: Load project repositories (done) | time=74ms
INFO: 1506 rcode with code coverage
```

Compiled against 12.1.

**Comment by lneumeier** [comment]
This error messages come from parsing 12.1 profiler outputs.

```
15:25:33.805 DEBUG: Parsing profiler output E:\Sonarqube\v501.69\profiler\profiler101384418.out
line 1336:4 token recognition error at: '-'
line 1337:4 token recognition error at: '-'
line 1338:3 token recognition error at: '-'
line 1339:3 token recognition error at: '-'
line 1340:3 token recognition error at: '-'
line 1341:3 token recognition error at: '-'
line 1342:3 token recognition error at: '-'
line 1343:2 token recognition error at: '-'
15:25:34.001 DEBUG: Parsing profiler output E:\Sonarqube\v501.69\profiler\profiler102068371.out
line 1159:3 token recognition error at: '-'
line 1160:3 token recognition error at: '-'
line 1161:3 token recognition error at: '-'
line 1162:2 token recognition error at: '-'
15:25:34.047 DEBUG: Parsing profiler output E:\Sonarqube\v501.69\profiler\profiler102174256.out
line 1301:4 token recognition error at: '-'
line 1302:4 token recognition error at: '-'
line 1303:4 token recognition error at: '-'
line 1304:4 token recognition error at: '-'
line 1305:3 token recognition error at: '-'
line 1306:3 token recognition error at: '-'
line 1307:3 token recognition error at: '-'
line 1308:3 token recognition error at: '-'
line 1309:3 token recognition error at: '-'
line 1310:2 token recognition error at: '-'
```

**Comment by gquerret** [comment]
Fixed in 2.6.3, see #753
